trunner/tools/phoenix.py

```diff
@@ -6,31 +6,31 @@
 import pexpect
 
 from trunner.harness import ProcessError
+from trunner.dut import PortNotFound
 from serial.tools import list_ports
 from contextlib import contextmanager
 from .common import add_output_to_exception
 
 
-def wait_for_vid_pid(vid: int, pid: int, timeout=0):
+def wait_for_vid_pid(vid: int, pid: int, timeout: int = 0, device: str = ""):
```

Comment:
I think that you shouldn't set a default value, coz in case that someone missed it you would get 2 spaces, we can require the name to be passed as an arg.